.github/workflows/data-product-example-2-poc.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: POC - zip and upload _example_2
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "data-products/_example_2/**"
+      - ".github/workflows/data-product-example-2-poc.yaml"
+
+jobs:
+  test:
+    uses: ./.github/workflows/data-product-example-2-test.yaml
+
+  upload:
+    name: Zip and upload
+    runs-on: ubuntu-latest
+    needs: test
+    permissions:
+      id-token: write # This is required for requesting the JWT
+      contents: read # This is required for actions/checkout
```

Comment:
```suggestion
```
I don't think this is explicitly required as this is a public repository, see https://github.com/ministryofjustice/data-platform-products/blob/main/.github/workflows/data-product-example-2-test.yaml